### PR TITLE
Mar3hc/feat/general keyword to send and receive uds service

### PR DESCRIPTION
**Pull request by trimai3001**
Add keyword Send UDS Request By Name for issue [https://github.com/test-fullautomation/robotframework-uds/issues/33], waiting for testing with DUT.

### CODE REVIEW

**Comment by trimai3001** [comment]
Hi @test-fullautomation,
This PR includes the solutions to the issue https://github.com/test-fullautomation/robotframework-uds/issues/31 and https://github.com/test-fullautomation/robotframework-uds/issues/33.
Thank you,
Tri

File: config/robotframework_aio/release_items_RobotFramework_UDS.json [hunk 1]
```diff
@@ -109,10 +109,23 @@ for more details.
...
+* ``Input Output Control By Name``
 * ``Write Data By Name`` features.
 * ``Get Encoded Request Message``
 * ``Get Decoded Response Message``
 
+Please refer to `RobotFramework_UDS documentation <https://github.com/test-fullautomation/robotframework-uds/blob/develop/RobotFramework_UDS/RobotFramework_UDS.pdf>`_
+for more details.
+"
+                                       ],
+                      "0.13.2." : [
```
Comment (test-fullautomation) [inline config/robotframework_aio/release_items_RobotFramework_UDS.json]: Hi Tri,
please change this to 0.13.1.
As I still wait for Son, I will merge it into 0.13.1.

Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Tri,
please check my finding.
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Tri,
this pull-request make #32 obsolete? Can I close #32?
Thank you,
 Thomas

**Pull request merged**